data/src/languages_update.py

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env python
+"""Updates languages.json
+
+This module takes the data/tsv directory as input and returns an
+updated version of languages.json, where the script entry for each
+language is updated to include every orthographic script present
+in each language's tsv file. Script entries are also updated
+to reflect such that script key entries match ISO 15924 aliases.
+"""
+
+import collections
+import json
+import operator
+import os
+
+from typing import Dict, DefaultDict, Optional
+
+import unicodedataplus  # type: ignore
+
+from data.src.codes import LANGUAGES_PATH, TSV_DIRECTORY_PATH  # type: ignore
```

Comment:
Surprised that this needs `type: ignore` since those are both obviously strings.